### PR TITLE
Remove protocol from web.archive.org URLs

### PR DESCRIPTION
**Pull request by vszakats**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [ ] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

### CODE REVIEW

**Comment by fxcoudert** [comment]
If these packages have no upstream anymore, I say we remove them.

**Comment by fxcoudert** [comment]
`cowsay` can migrate to https://github.com/tnalpgge/rank-amateur-cowsay (after all we already get the source from there)

**Comment by vszakats** [comment]
Moved `cowsay` to #11337.

File: Formula/libagg.rb [hunk 1]
```diff
@@ -1,7 +1,7 @@
 class Libagg < Formula
   desc "High fidelity 2D graphics library for C++"
-  homepage "https://web.archive.org/web/20150812005010/http://www.antigrain.com/"
```
Comment (fxcoudert) [inline Formula/libagg.rb]: http://www.antigrain.com works

File: Formula/rogue.rb [hunk 2]
```diff
@@ -1,6 +1,6 @@
 class Rogue < Formula
   desc "Dungeon crawling video game"
-  homepage "https://web.archive.org/web/20160604020207/http://rogue.rogueforge.net/"
+  homepage "https://web.archive.org/web/20160604020207/rogue.rogueforge.net/"
```
Comment (fxcoudert) [inline Formula/rogue.rb]: `rogue` we should try to find a solution, if only for historical reasons :)

**Review by fxcoudert — COMMENTED** [review]
Other formulas I have opened PR for migration to boneyard

File: Formula/rogue.rb (hunk 2 above)
Comment (vszakats) [inline Formula/rogue.rb]: It still has a sf.net homepage: <https://sourceforge.net/projects/roguelike/>

File: Formula/libagg.rb (hunk 1 above)
Comment (vszakats) [inline Formula/libagg.rb]: Updated!

Comment (vszakats) [inline Formula/libagg.rb]: Updated!

**Comment by vszakats** [comment]
Removed all boneyarded formulae.

**Comment by vszakats** [comment]
Jenkins issue on El Cap/Sierra.

**Comment by vszakats** [comment]
A Jenking nudge would be appreciate here.

**Comment by fxcoudert** [comment]
Boneyard migration was not well received, sorry to add to your work here.

**Comment by vszakats** [comment]
No worries, I've kept the scripts, so I could redo the deleted commits quickly.

**Comment by fxcoudert** [comment]
Thanks again @vszakats !

**Pull request merged**